extensions/blocks/slideshow/style.scss

```diff
@@ -1,10 +1,16 @@
 @import '../../shared/styles/gutenberg-colors.scss';
 @import '../../shared/styles/jetpack-variables.scss';
 
+$break-small: 600px;
```

Comment:
You can import `../..//shared/styles/gutenberg-variables.scss` to have the same variable from there.

Reply:
Oh cool, I didn't know 😊

Reply:
Done in c7b0f22